tests/data/packages/LocalExtrasDuplicated/setup.py

```diff
@@ -0,0 +1,30 @@
+import os
+from setuptools import setup, find_packages
+
+
+def path_to_url(path):
+    """
+    Convert a path to URI. The path will be made absolute and
+    will not have quoted path parts.
+    """
+    path = os.path.normpath(os.path.abspath(path))
+    drive, path = os.path.splitdrive(path)
+    filepath = path.split(os.path.sep)
+    url = '/'.join(filepath)
+    if drive:
+        return 'file:///' + drive + url
+    return 'file://' +url
+
+
+HERE = os.path.dirname(__file__)
+DEP_PATH = os.path.join(HERE, '..', '..', 'indexes', 'simple', 'simple')
+DEP_URL = path_to_url(DEP_PATH)
+
+setup(
+    name='LocalExtrasDuplicated',
+    version='0.0.1',
+    packages=find_packages(),
+    extras_require={ 'shrubbery': ['LocalExtras[baz]']},
+    install_requires=['LocalExtras[bar]'],
+    dependency_links=[DEP_URL]
```

Comment:
You could just add `simple` as a dependency and use `script.pip_install_local` in the test, using the package names.

Reply:
I reused the already existing LocalExtras package. So, to change that, here would be:
```python
# No need for path_to_url, HERE, DEP_PATH or DEP_URL

setup(
    ...
    install_requires=['simple', 'LocalExtras[Bar]'],
    ...
)
```
And then in the test something like:
```python
result = script.pip_local_install('--no-index', '-f', data.find_links, to_install + '[shrubbery]')
```
?

(it's been quite a long time, so my memory is a bit rusty)

Reply:
Yes. Off the top of my head, the test would be:

```py
result = script.pip_local_install(to_install + '[shrubbery]')
```